rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -82,6 +95,19 @@ public Sw360Module() {
             setMixInAnnotation(ComponentDTO.class, Sw360Module.ComponentDTOMixin.class);
             setMixInAnnotation(Package.class, Sw360Module.PackageMixin.class);
             setMixInAnnotation(Release.class, Sw360Module.ReleaseMixin.class);
+            setMixInAnnotation(SPDXDocument.class, Sw360Module.SPDXDocumentMixin.class);
+            setMixInAnnotation(DocumentCreationInformation.class, Sw360Module.DocumentCreationInformationMixin.class);
+            setMixInAnnotation(PackageInformation.class, Sw360Module.PackageInformationMixin.class);
+            setMixInAnnotation(CheckSum.class, Sw360Module.CheckSumMixin.class);
+            setMixInAnnotation(ExternalDocumentReferences.class, Sw360Module.ExternalDocumentReferencesMixin.class);
+            setMixInAnnotation(Creator.class, Sw360Module.CreatorMixin.class);
+            setMixInAnnotation(PackageVerificationCode.class, Sw360Module.PackageVerificationCodeMixin.class);
+            setMixInAnnotation(ExternalReference.class, Sw360Module.ExternalReferenceMixin.class);
+            setMixInAnnotation(SnippetRange.class, Sw360Module.SnippetRangeMixin.class);
+            setMixInAnnotation(Annotations.class, Sw360Module.AnnotationsMixin.class);
+            setMixInAnnotation(RelationshipsBetweenSPDXElements.class, Sw360Module.RelationshipsBetweenSPDXElementsMixin.class);
+            setMixInAnnotation(SnippetInformation.class, Sw360Module.SnippetInformationMixin.class);
+            setMixInAnnotation(OtherLicensingInformationDetected.class, Sw360Module.OtherLicensingInformationDetectedMixin.class);
```

Comment:
Please add the same mixins for `SpringDocUtils.getConfig().replaceWithClass()` for OpenAPI docs.

Reply:
Added them for SpringDocUtils